game.go

```diff
@@ -115,13 +115,15 @@ func nextGameState(state GameState) GameState {
 
 type Game struct {
 	GameState
-	ID           string    `json:"id"`
-	CreatedAt    time.Time `json:"created_at"`
-	UpdatedAt    time.Time `json:"updated_at"`
-	StartingTeam Team      `json:"starting_team"`
-	WinningTeam  *Team     `json:"winning_team,omitempty"`
-	Words        []string  `json:"words"`
-	Layout       []Team    `json:"layout"`
+	ID            string    `json:"id"`
+	CreatedAt     time.Time `json:"created_at"`
+	UpdatedAt     time.Time `json:"updated_at"`
+	StartingTeam  Team      `json:"starting_team"`
+	WinningTeam   *Team     `json:"winning_team,omitempty"`
+	Words         []string  `json:"words"`
+	Layout        []Team    `json:"layout"`
+	TimerSettings []int32   `json:"timer_settings"`
+	EndTime       time.Time `json:"end_time,omitempty"`
```

Comment:
```suggestion
	RoundStartedAt       time.Time `json:"round_started_at,omitempty"`
```

How about a `RoundStartedAt` timestamp rather then `EndTime`? Then the client can compute the end time on their own. It also still has meaning when you're not using a timer, whereas `EndTime` does not.

(The past tense and 'At' is to match `CreatedAt` and `UpdatedAt`)